xtask/src/main.rs

```diff
@@ -130,3 +137,31 @@ fn project_root() -> PathBuf {
 fn dist_dir() -> PathBuf {
     project_root().join("target/dist")
 }
+
+fn build_docker(all: bool, build_images: Vec<String>) -> Result<()> {
+    let mut images = build_images;
+    if all {
+        images = vec![String::from("iroh-one"), String::from("iroh-p2p")];
+    }
+
+    for image in images {
+        let status = Command::new("docker")
+            .current_dir(project_root())
+            .args([
+                "build",
+                "-t",
+                format!("{}:distroless", image).as_str(),
```

Comment:
this tags images in the form iroh-one:distroless... that's probably wrong

Reply:
yeah, for starters let's do `:commit_hash` and then builders can also tag with `latest`